experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx

```diff
@@ -86,6 +88,20 @@ export const MOBILE_APPLICATIONS = [
   NimbusExperimentApplicationEnum.FOCUS_IOS,
 ];
 
+const APP_NAME_TO_SLUG_MAP: Record<NimbusExperimentApplicationEnum, string> = {
+  DESKTOP: "firefox_desktop",
+  FENIX: "fenix",
+  IOS: "firefox_ios",
+  DEMO_APP: "NOT_SUPPORTED",
+  FOCUS_ANDROID: "focus_android",
+  FOCUS_IOS: "focus_ios",
+  FXA: "NOT_SUPPORTED",
+  KLAR_ANDROID: "klar_android",
+  KLAR_IOS: "klar_ios",
+  MONITOR: "monitor_cirrus",
+  VPN: "NOT_SUPPORTED",
+};
```

Comment:
This kind of thing should really come from the backend instead of being hard coded here.  I'd say it doesn't matter if we rip out the old frontend soon anyways but I suspect we will keep some of the data specific parts for at least some period of time so we should do these parts properly.  This information can all be gathered in the backend as constants and shared through the config endpoint.